src/isr.cpp

```diff
@@ -174,10 +174,8 @@ void isr() {
 		}
 	}
 	// glides
-	// in monophonic mode, we glide only while the old note is still held down.
-	bool skip_glide = !preset_data.is_polyphonic() && voice_state.n_held_keys() <= 1;
 	for (int i = 0; i < 6; i++) {
-		glide[i].glide_tick(skip_glide ? 0 : preset_data.voice[voice_index[i]].glide);
+		glide[i].glide_tick(voice_state.shall_glide(i) ? preset_data.voice[voice_index[i]].glide : 0);
```

Comment:
@twisted-electrons look how while we let the `VoiceState` *decide* whether or not to glide, still the `isr.cpp` file is in control to *actually* instruct our `Glide` objects to glide slowly or infinitely-quickly.

Reply:
also note that this needs to be tested on real hardware; this timer ISR handler is on a pretty tight time budget and we changed the code to make 6 decisions instead of just one. Hopefully this doesn't overrun our time budget (if so, some more changes are needed.)